src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private int range = 40;
```

Comment:
The range variable should be a constant since it represents a fixed range of values for figures sizes. Consider declaring it as a public static final variable and use uppercase letters according to the Java naming conventions.